src/layout/Header.tsx

```diff
@@ -1 +1,61 @@
-export const Header = () => <div className="font-bold">Headerrr</div>;
+import { SocialIcon } from './components/SocialIcon';
+import { NavHeaderLink } from './components/NavLinks';
+import { useState } from 'react';
+import { faViber, faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
+import Logo from '../images/VilaMarina.png';
+
+export const Header = () => {
+   const [menuOpen, setMenuOpen] = useState(false);
+
+   const toggleMenu = () => {
+      setMenuOpen(!menuOpen);
+   };
```

Comment:
We usually use this approach, remind me to tell you why.

```suggestion
   const toggleMenu = () => setMenuOpen(mo => !mo);
```